frontend/src/metabase/dashboard/reducers.js

```diff
@@ -518,10 +518,7 @@ const autoApplyFilters = handleActions(
 );
 
 const theme = createReducer(INITIAL_DASHBOARD_STATE.theme, builder => {
-  builder.addCase(
-    SET_DISPLAY_THEME,
-    (state, { payload }) => payload ?? "light",
-  );
+  builder.addCase(SET_DISPLAY_THEME, (state, { payload }) => payload || null);
```

Comment:
Very subtle bug caught by E2E. It was adding `#theme=light` to some route's url, so best to make it nullable for now to keep existing behaviour.